flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks/OrionMySQLSink.java

```diff
@@ -66,7 +66,7 @@
      */
     public OrionMySQLSink() {
         super();
-        logger = CygnusLogger.getLogger(OrionHDFSSink.class);
+        logger = CygnusLogger.getLogger(OrionMySQLSink.class);
```

Comment:
This seems to be a bugfix... It has any impact? Or it happened that the logger for HDFS sink and SQL were the same so it is a small bugfix?